Games/types/Mafia/roles/cards/ItemTransfer.js

```diff
@@ -0,0 +1,48 @@
+const Card = require("../../Card");
+const { PRIORITY_ITEM_GIVER_DEFAULT } = require("../../const/Priority");
+
+module.exports = class ItemTransfer extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.meetings = {
+            "Steal From": {
+                states: ["Night"],
+                flags: ["voting"],
+                targets: { include: ["alive"], exclude: ["dead", "self"] },
+                action: {
+                    labels: ["stealItem"],
+                    priority: PRIORITY_ITEM_GIVER_DEFAULT - 5,
```

Comment:
fyi it's -5 because asso is already -1, i don't want to make asso gun stealable the night it is given.

If you want to keep it -1 it's fine too